docs/en/common/module/coffee-module-etcd.adoc

```diff
@@ -49,6 +49,14 @@ even if the ETCD does not contain it,
 so we can reduce a lot of repetitive queries.
 Cache is a thread safe singleton,
 it is possible to clear it by calling `EtcdConfigSourceCache.instance().clear()`.
+It is possible to disable a ConfigSource with configuration, e.g.:
++
+----
+hu.icellmobilsoft.coffee.module.etcd.producer.CachedEtcdConfigSource.enabled: false
+----
+There are two ways to set this configuration based on the return value of the following methods:
+. `ConfigurableConfigSource.getClassEnableKey()`
+. `ConfigurableConfigSource.getConfigSourceNameKey()`
```

Comment:
- így olyan mintha csak a CachedEtcdConfigSource-ra vonatkozna, viszont minden olyanra vonatkozik, ami a `ConfigurableConfigSource`-ból származik. Külön kiemelném ezt a kikapcsolhatóság témát és több példát adnék.
- Jó lenne példa a rövidebb kulcsra is a `ConfigurableConfigSource.getConfigSourceNameKey()` által, pl: `CachedEtcdConfigSource.enabled: false`
- `docs/hu/` alá már nem kell a magyar?
- `release-notes`-ban is érdemes lenne megemlíteni, hogy a dokumentáció kiegészült ezzel a hiányos résszel.